app/scripts/framework/service/meta-service.js

```diff
@@ -162,6 +162,9 @@ org.ekstep.services.metaService = new (org.ekstep.services.iService.extend({
 	},
 
 	getVideoLicense: function (data, callback) {
-		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('videoLicenseURL', '/asset/v3/license/validate'), data, this.requestHeaders, callback)
+		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('videoLicenseURL', '/asset/v3/validate?field=license'), data, this.requestHeaders, callback)
```

Comment:
You can pass `field` also in for function same function will be useful later if they add one more fieldType

Reply:
Yes, changes are done and pushed